posthog/api/query.py

```diff
@@ -250,6 +253,18 @@ def process_query(
         )
         serializer.is_valid(raise_exception=True)
         return get_session_events(serializer) or {}
+    elif query_kind == "WebTopSourcesQuery":
+        refresh_requested = refresh_requested_by_client(request) if request else False
+        top_sources_query_runner = TopSourcesQueryRunner(query_json, team)
+        return _unwrap_pydantic_dict(top_sources_query_runner.run(refresh_requested=refresh_requested))
+    elif query_kind == "WebTopClicksQuery":
+        refresh_requested = refresh_requested_by_client(request) if request else False
+        top_clicks_query_runner = TopClicksQueryRunner(query_json, team)
+        return _unwrap_pydantic_dict(top_clicks_query_runner.run(refresh_requested=refresh_requested))
```

Comment:
Let's add the extra 3 letters and make these consistent/explicit. I don't think it's worth introducing an inconsistency here.

```suggestion
    elif query_kind == "WebTopClicksQuery":
        refresh_requested = refresh_requested_by_client(request) if request else False
        web_top_clicks_query_runner = WebTopClicksQueryRunner(query_json, team)
        return _unwrap_pydantic_dict(web_top_clicks_query_runner.run(refresh_requested=refresh_requested))
```

Reply:
Fixed